spec/hashie/utils_spec.rb

```diff
@@ -0,0 +1,25 @@
+require 'spec_helper'
+
+def a_method_to_match_against
```

Comment:
Generally I stay away from global methods, I would move it inside `RSpec.describe` which I think namespaces it, or namespace it explicitly. It's ok for now.